.github/workflows/test_python.yaml

```diff
@@ -116,14 +103,22 @@ jobs:
           git reset --hard
       # Sphinx 6.x #####################################################################
       - name: Test Python ${{ matrix.python-version }} / sphinx 6.x
-        # sphinx 6.x requires python 3.8+
-        if: ${{ matrix.python-version != '3.7' }}
         env:
           # Test sphinx 6.x.
           SPHINX_VERSION: '>=6,<7'
         run: |
           tox -e py -- --cov-report xml:coverage.xml --cov
       - name: Upload Code Coverage for Python ${{ matrix.python-version }} / sphinx 6.x
-        if: ${{ matrix.python-version != '3.7' }}
         run: |
           codecov --required -X gcov -f coverage.xml --name "[GHA] ${{ matrix.os }}-py${{ matrix.python-version }}-sphinx6.x"
+      # Sphinx 7.x #####################################################################
+      - name: Test Python ${{ matrix.python-version }} / sphinx 6.x
+        env:
+          # Test sphinx 7.x.
+          SPHINX_VERSION: '>=7,<8'
```

Comment:
breathe is not compatible with 7.2 yet. (https://github.com/svenevs/exhale/issues/203#issuecomment-1773060684)
So test must be  `SPHINX_VERSION: '>=7,<7.2'`

Reply:
Perfect, thank you!